fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -172,7 +174,31 @@ def _call_oracle(self, lambda_vec):
             estimator = clone(estimator=self.estimator, safe=False)
 
         oracle_call_start_time = time()
-        estimator.fit(self.constraints.X, redY, **{self.sample_weight_name: redW})
+
+        # The line that is being replaced
```

Comment:
Thanks for providing the context! Instead of putting it in the code, do you mind pasting it in the PR description with triple backticks/ ` ? Then it render like this:
```
this is code
```